stginga/plugins/local_plugin_mixin.py

```diff
@@ -26,20 +26,27 @@
 class HelpMixin(object):
     def help(self):
         """Display online help for the plugin."""
-        if not self.fv.gpmon.has_plugin('WBrowser'):
-            self._help_docstring()
-            return
+        if self.fv.gpmon.has_plugin('WBrowser') and Widgets.has_webkit:
+            # ginga < v5.x
+            self.fv.start_global_plugin('WBrowser')
+
+            # need to let GUI finish processing, it seems
+            self.fv.update_pending()
 
-        self.fv.start_global_plugin('WBrowser')
+            obj = self.fv.gpmon.get_plugin('WBrowser')
 
-        # need to let GUI finish processing, it seems
-        self.fv.update_pending()
+            # Unlike Ginga, we do not attempt to download offline doc
+            # but just point to online doc directly.
+            obj.browse(self.help_url)
+            return
 
-        obj = self.fv.gpmon.get_plugin('WBrowser')
+        if not hasattr(self.fv, 'help_plugin'):
```

Comment:
With Ginga 4, because I don't have `python-webkit`, I hit this line but only if I do not have docutils installed. I noticed it because the terminal complains about `'publish_string' is not defined`.

However, when I install docutils, I do not hit this, instead Ginga now tries to fire up WBrowser and complains that I do not have webkit. 🤯

Reply:
Right, the web browser widget does not seem to be installed by default when you install qt, and unfortunately, in Ginga 4 this does not disable `WBrowser`. Instead, when you start `WBrowser`, it tells you to install webkit to get things working. (Qt has changed the name of their version of webkit and in Qt 5 it's name is `qtwebengine` AFAIK). 

Without a working `WBrowser`, in Ginga 4 it will attempt to fall back to `_help_docstring()` which in Ginga 4 will attempt to use `docutils` (another thing I am getting rid of). I've updated this PR so that it will use an `stginga` version of `_help_docstring`, which will punt directly to showing the RST text.  Give it a try and LMK how it goes.